string utterance
uint8 confidence
string source
string extra
